### DISCUSSION TITLE
Hide scrollbar with display:none

### DISCUSSION THREAD

**Discussion by miguelbinue** [idiotWu/smooth-scrollbar#528] [Show and tell]
### Motivation

How could I hide the scrollbar, but maintaining the functionalities on the website?

### Proposal

I added this to my css file:

.scrollbar-track {
  display: none !important;
}

It disables the styles of the scrollbar.
I'm new in coding and smooth-scrollbar as well, but I hope this could help newbies.

**Comment by sadeghbarati** [comment]
I don't see much of this use case, only saw some awwwards sites which use smooth scroll libs without scrollbar

**Comment by dolphin-wood** [comment]
If you want to completely remove the scrollbar tracks, try:

```js
scrollbar.track.xAxis.element.remove();
scrollbar.track.yAxis.element.remove();
```
